smbclient/_os.py

```diff
@@ -55,7 +55,7 @@
 from smbprotocol.ioctl import (
     CtlCode,
     IOCTLFlags,
-)
+    SMB2SrvCopyChunk, SMB2SrvCopyChunkResponse, SMB2SrvCopyChunkCopy, SMB2SrvRequestResumeKey)
```

Comment:
Best to keep the same style as above and put an import on a newline. This is mostly to make it easier to alphabetise and add/remove imports without worrying about the line length

Reply:
done